### DISCUSSION TITLE
[Feature] Read/write Picasa's face exif metadata

### DISCUSSION THREAD

**Discussion by rodrigopolo** [immich-app/immich#10466] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Picasa was replaced by Google Photos, Picasa was the best Photo app at the moment, and one of the killer features was its face detection, but while migrating from one computer to another, sometimes the database got corrupted, and re-scanning and naming again was tedious, that is why Picasa implemented the option to store the necessary metadata of each face detected into the images:

![Screenshot 2024-06-18 at 11 54 04 AM](https://github.com/immich-app/immich/assets/1511174/ad411df5-9b87-488f-b56a-05d7519c0bdd)

This metadata can be viewed with exiftool, here is an example of the tagged faces in one image, it is self explanatory:
exiftool -s -G image.jpg
```
[XMP]           RegionAppliedToDimensionsW      : 8192
[XMP]           RegionAppliedToDimensionsH      : 5464
[XMP]           RegionAppliedToDimensionsUnit   : pixel
[XMP]           RegionName                      : Hugo, Paco, Luis, Rodrigo, Carlos
[XMP]           RegionType                      : Face, Face, Face, Face, Face
[XMP]           RegionAreaX                     : 0.444641, 0.594666, 0.376892, 0.286865, 0.510315
[XMP]           RegionAreaY                     : 0.169015, 0.327233, 0.452233, 0.401171, 0.453148
[XMP]           RegionAreaW                     : 0.10437, 0.111938, 0.109985, 0.101318, 0.134399
[XMP]           RegionAreaH                     : 0.188324, 0.201318, 0.198389, 0.18265, 0.241581
[XMP]           RegionAreaUnit                  : normalized, normalized, normalized, normalized, normalized
```
Immich could read this metadata to aid its face detection and labeling for those migrating from Picasa to Immich, believe me, there are plenty of people who still uses Picasa refusing to go into "the cloud", also, it would be great to have the option to also write this metadata directly into the original files.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4348
